pyvista/core/objects.py

```diff
@@ -375,6 +375,24 @@ def repeat(self):
     def repeat(self, flag):
         self.SetRepeat(flag)
 
+    @property
+    def cube_map(self, flag):
```

Comment:
Do properties with input arguments actually work? Same goes for the `setter`, where the second arg isn't unused.

Reply:
No, they don't.

Reply:
Fixed